css/demo.css

```diff
@@ -15,27 +16,27 @@ body {
   margin-block-start: var(--page-margin);
 }
 
-#banner-logo {
-  &:link,
-  &:visited {
-    align-self: start;
-    display: block;
-    grid-area: logo;
-    transition: transform 300ms cubic-bezier(0.175, 0.885, 0.32, 1.275);
-    transform-origin: top center;
-    z-index: 1;
-  }
+#banner-logo:hover,
+#banner-logo:focus,
+#banner-logo:active {
+  transform: scale(1.05);
+}
 
-  &:hover,
-  &:focus,
-  &:active {
-    transform: scale(1.05);
-  }
+#banner-logo [data-logo] {
+  block-size: auto;
+  height: auto;
+  inline-size: 100%;
+  width: 100%;
```

Comment:
@dvdherron I didn't comment on these as I think we do actually want the more "modern" property last as it will be ignored if a browser doesn't understand it. So the order:

height
block-size
width
inline-size 

is probably what we want, but other than that,  I'd say this is good to merge!

Reply:
good catch. fixed in https://github.com/oddbird/popover-polyfill/pull/236/commits/5ebfc647404ed7cfb6db32bdf44d6ff5ce701c42 so I think this is good to merge!